src/brogue/IncludeGlobals.h

```diff
@@ -232,3 +233,8 @@ extern const char armorRunicNames[NUMBER_ARMOR_ENCHANT_KINDS][30];
 extern const char monsterBehaviorFlagDescriptions[32][COLS];
 extern const char monsterAbilityFlagDescriptions[32][COLS];
 extern const char monsterBookkeepingFlagDescriptions[32][COLS];
+
+// from the platform code
+extern boolean serverMode;
+extern boolean hasGraphics;
+extern boolean showGraphics;
```

Comment:
I quite like defining `hasGraphics` in the platform code, but would showGraphics better be defined where it is used, i.e. top of `IO.c`. Given that turning graphics on and off is conceptually part of the main code, not a platform-specific.

Reply:
Yes good idea, I need to reshuffle this